domains/devmatei.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "DevMatei",
+           "email": "",
+           "discord": "784443338627612673",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Xfkro-38gTXoPdcrF-mGP-0gpbfMqsWBqeYOyYVKKsmt3ImwKpLmeMfsMeYxXLzyjVCRhnSUPGYYvoqg773HsJGh8dLnyuCI7SgiolIqnb2pS7t3xyxNX2sq0QBPPqlGLzDGhRYMVjIq02aNIksdh3XiJ3vsS36yVxJ4bOb4gG-t4y6hqmxHLICK7wPwG3zVQcqse63QtEyNdPpmDNbzmcmp3pxDzD-cmrX8FphYTcABbL7pphEZyZNMTtVaR-mfKafVmd2pmE_L-J3EKNSzLJj-8HSFx5N4KN36uWs01sBKHWRWz5Co6YRvT3eQxgu9WYuocLlFrZ8aPqQcsCkXGg.YVkATTXNM6fkDeg4Ew01JA.YXGkCx-tdRkCp2th5RJLnEsOUSmisdUKE4KbpcxG9ZVVodzbkbDXwLwjqTntT_5xqe84ixhZfRCb-TAGAYkWyopaG3EpnWtWYkLNB43Vpfo.8xLIYHMxurOHww7iUeAqMQ"
+        },
+    
+        "record": {
+            "TXT": "_vercel.devmatei.is-a.dev"
```

Comment:
TXT does nothing, if you want to use Vercel then please check the docs: https://www.is-a.dev/docs/vercel/